packages/console/package.json

```diff
@@ -8,10 +8,11 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "precommit": "lint-staged",
-    "start": "parcel src/index.html",
-    "dev": "PORT=5002 parcel src/index.html --public-url /console --no-hmr --no-cache",
+    "prepare": "copyfiles -u 1 public/**/*.* dist",
```

Comment:
prepare will be autorun by the pnpm install script. Will it cause redundant and unexpected actions?

Reply:
Well, then we can rename it to `copyfiles`.